src/QsCompiler/CommandLineTool/Options.cs

```diff
@@ -136,32 +187,31 @@ internal void UpdateSetIndependentSettings(Options updates, bool overwriteNonDef
             this.References = (this.References ?? Array.Empty<string>()).Concat(updates.References ?? Array.Empty<string>());
         }
 
-
-        // routines related to logging 
+        // routines related to logging
 
         /// <summary>
-        /// If a logger is given, logs the options as CommandLineArguments Information before returning the printed string. 
+        /// If a logger is given, logs the options as CommandLineArguments Information before returning the printed string.
         /// </summary>
         public string[] Print(ILogger logger = null)
         {
-            string value(PropertyInfo p)
+            string Value(PropertyInfo p)
```

Comment:
How come private members are lower case but local functions are upper case? It would seem more consistent to me if those where lower case then, too.

Reply:
It's only private fields that are lowercase - private methods (and even private properties) are capitalized.